addons/website/data/website_data.xml

```diff
@@ -44,88 +44,60 @@
-                                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact6">
-                                                            <span class="s_website_form_label_content">Question</span>
-                                                            <span class="s_website_form_mark"> *</span>
-                                                        </label>
-                                                        <div class="col-sm">
-                                                            <textarea id="contact6" class="form-control s_website_form_input" name="description" required=""></textarea>
-                                                        </div>
-                                                    </div>
+                                                    <label class="s_website_form_label" style="width: 200px" for="contact6">
+                                                        <span class="s_website_form_label_content">Question</span>
+                                                        <span class="s_website_form_mark"> *</span>
+                                                    </label>
+                                                    <textarea id="contact6" class="form-control s_website_form_input" name="description" required="" rows="8"></textarea>
                                                 </div>
                                                 <div class="mb-0 py-2 col-12 s_website_form_field s_website_form_dnone">
-                                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact7">
-                                                            <span class="s_website_form_label_content">Email To</span>
-                                                        </label>
-                                                        <div class="col-sm">
-                                                            <input id="contact7" type="hidden" class="form-control s_website_form_input" name="email_to"/>
-                                                        </div>
-                                                    </div>
+                                                    <label class="s_website_form_label" style="width: 200px" for="contact7">
+                                                        <span class="s_website_form_label_content">Email To</span>
+                                                    </label>
+                                                    <input id="contact7" type="hidden" class="form-control s_website_form_input" name="email_to"/>
```

Comment:
Note that it doesn't break anything, but in fact we cannot edit this field with the option, so there was no need to modify it in the template :open_mouth: Maybe keep it like before ? :slightly_smiling_face: 
But note that you can still change the `mb-0 py-2` by `mb-3` as it is the default when dropping a form :slightly_smiling_face: